### ISSUE TITLE
New Feature Required - During RPM update configuration files are overwriting- Backup of existing files feature required

### ISSUE THREAD

**Issue by Manoharan-NMS** [signal18/replication-manager#430]
During every update the below 2 files & newly created cluster conf files are not taken backup. Its overwriting always.

```
/etc/replication-manager/config.toml
/etc/replication-manager/cluster.d/cluster1.toml
```

Kindly provide this feature. It may be useful for production environment.

RPM Reference Link: https://ci.signal18.io/mrm/builds/tags/v2.2.20/replication-manager-osc-2.2.20-1.x86_64.rpm

**Comment by tanji** [comment]
Thanks for reporting, I will check

**Comment by Manoharan-NMS** [comment]
> Thanks for reporting, I will check

Thanks for your response

**Comment by tanji** [comment]
fixed in c05110f798314310ba6b29916b2f54a7530a6821, will be included in next packages

**Comment by Manoharan-NMS** [comment]
> fixed in [c05110f](https://github.com/signal18/replication-manager/commit/c05110f798314310ba6b29916b2f54a7530a6821), will be included in next packages

Thanks for your fix.

In case new parameters are added in configuration file, is it handled. During that time also, file is retained, user will not be able to get new parameter related information.

Hope its handled.
